crates/astria-conductor/src/conductor.rs

```diff
@@ -152,13 +159,42 @@ impl Conductor {
             let (shutdown_tx, shutdown_rx) = oneshot::channel();
             shutdown_channels.insert(Self::DATA_AVAILABILITY, shutdown_tx);
             let block_verifier = BlockVerifier::new(sequencer_client_pool.clone());
+
+            // Sequencer namespace is defined by the chain id of attached sequencer node
+            // which can be fetched from any block header.
+            let sequencer_namespace = {
+                use sequencer_client::SequencerClientExt as _;
+
+                let client = sequencer_client_pool
+                    .get()
+                    .await
+                    .wrap_err("failed to get a sequencer client from the pool")?;
+                let block = tryhard::retry_fn(|| client.latest_sequencer_block())
```

Comment:
This has 10 retries but no backoff strategy. Is that intentional? This basically turns this into 10 requests that get fired off immediately one after the other.

Reply:
Changed to exponential.